code/js/controllers/DuckburgRadioController.js

```diff
@@ -0,0 +1,18 @@
+;(function() {
+  "use strict";
+
+  var BaseController = require("BaseController");
+
+  new BaseController({
+    siteName: "Duckburg Radio",
+
+    buttonSwitch: true,
```

Comment:
`buttonSwitch` should only be set when a player has two different buttons (ie. html elements) for play and pause.

I'm guessing that since you only have a `playPause` selector (and not individual `play` and `pause` selectors) that `buttonSwitch` is not needed for this controller.